server/routes/users.js

```diff
@@ -0,0 +1,37 @@
+const express = require("express");
+const router = express.Router();
+const mongoose = require("mongoose");
+
+let databaseConnection = "Waiting for Database response...";
+
+router.get("/", (req, res, next) => {
+    res.send(databaseConnection);
+});
+
+mongoose.connect("mongodb://localhost:27017/mentions", { useNewUrlParser: true  });
```

Comment:
I'd rather the url is kept in the .env file

Reply:
I changed to to use the env file